Mathlib/LinearAlgebra/FiniteDimensional.lean

```diff
@@ -372,6 +372,7 @@ theorem eq_top_of_finrank_eq [FiniteDimensional K V] {S : Submodule K V}
   rw [bS_eq, Basis.coe_ofVectorSpace, Subtype.range_coe] at this
   rw [this, map_top (Submodule.subtype S), range_subtype]
 #align finite_dimensional.eq_top_of_finrank_eq FiniteDimensional.eq_top_of_finrank_eq
+#align submodule.eq_top_of_finrank_eq FiniteDimensional.eq_top_of_finrank_eq
```

Comment:
I think the `Submodule.eq_top_of_finrank_eq` name is a little better, as the same statement could (or maybe even does) exist for `Subalgebra`